### PR TITLE
About

### PR DESCRIPTION
**Pull request by IvanBartos**
reworks the "about" section
- adds profile images for interns
- updates subheadings for several members
- adds two subsections: "spominame" and "stazisti"
- sorts all members to respective subsections

### CODE REVIEW

File: views/about.erb [hunk 1]
```diff
@@ -160,6 +146,39 @@
...
+              <h3>Michal Truban</h3>
+              <span>Zakladajúci člen</span>
+              <!-- <p class="perex">Majiteľ hostingovej spoločnosti <a href="https://www.websupport.sk/">WebSupport, s.r.o.</a>, v roku 2015 bol vyhlásený za IT osobnosť roka na Slovensku.</p>
+              <a href="mailto:michal.truban@slovensko.digital">michal.truban@slovensko.digital</a> -->
+            </div>
+          </div>
+
+          <div class="col-sm-4">
+            <div class="content-wrap">
+              <img src="/img/onas/michal_cheben.jpg" alt="Michal Chebeň">
```
Comment (jsuchal) [inline views/about.erb]: ň preč

File: views/about.erb [hunk 2]
```diff
@@ -160,6 +146,39 @@
             </div>
           </div>
         </div>
+        <h3 class="dark-heading"><strong>Spomíname</strong></h3>
+        <div class="row team-wider">
+
+          <div class="col-sm-4">
+            <div class="content-wrap">
+              <img src="/img/onas/michal_truban.jpg" alt="Micahl truban">
```
Comment (jsuchal) [inline views/about.erb]: preklep

File: views/about.erb [hunk 3]
```diff
@@ -160,6 +146,39 @@
             </div>
           </div>
         </div>
+        <h3 class="dark-heading"><strong>Spomíname</strong></h3>
```
Comment (jsuchal) [inline views/about.erb]: Ocenujem vtip ale neviem ci toto prejde cez jana hargasa. Zakladajuci clenovia stale su.

File: views/about.erb [hunk 4]
```diff
@@ -143,6 +104,31 @@
...
+              <img src="/img/onas/lubos_pernis.jpg" alt="Luboš Perniš">
+              <h3>Luboš Perniš</h3>
+              <span>Rozvoj zahraničnej spolupráce</span>
+            </div>
+          </div>
+          <div class="col-sm-4">
+            <div class="content-wrap">
+              <img src="/img/onas/alexandra_urbanikova.jpg" alt="Alexandra Urbaníkova">
+              <h3>Alexandra Urbaníkova</h3>
+              <span>Rozvoj zahraničnej spolupráce</span>
```
Comment (jsuchal) [inline views/about.erb]: Ale pravdu!

File: views/about.erb [hunk 5]
```diff
@@ -151,7 +151,7 @@
 
           <div class="col-sm-4">
             <div class="content-wrap">
-              <img src="/img/onas/michal_truban.jpg" alt="Micahl truban">
+              <img src="/img/onas/michal_truban.jpg" alt="Micahl Truban">
```
Comment (jsuchal) [inline views/about.erb]: Este stale preklep

Comment (IvanBartos) [inline views/about.erb]: slepý som - az teraz som si vsimol

**Pull request merged**